tests/ballerina-integration-test/src/test/java/org/ballerinalang/test/auth/AuthzConfigPatternTest.java

```diff
@@ -0,0 +1,125 @@
+/*
+ *  Copyright (c) 2018, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+ *
+ *  WSO2 Inc. licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except
+ *  in compliance with the License.
+ *  You may obtain a copy of the License at
+ *
+ *  http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ *  KIND, either express or implied.  See the License for the
+ *  specific language governing permissions and limitations
+ *  under the License.
+ */
+
+package org.ballerinalang.test.auth;
+
+import org.ballerinalang.test.util.HttpResponse;
+import org.ballerinalang.test.util.HttpsClientRequest;
+import org.testng.annotations.Test;
+
+import java.util.HashMap;
+import java.util.Map;
+
+/**
+ * Test cases for authorization config pattern test scenarios.
+ */
+@Test(groups = "auth-test")
+public class AuthzConfigPatternTest extends AuthBaseTest {
+
+    private final int servicePort = 9115;
+
+    @Test(description = "Test valid user for pattern 1")
```

Comment:
Do we define pattern1 to pattern5 somewhere. Missed it. Can we define what each pattern is in a comment?

Reply:
Yeah. Added in java docs.